lib/etl/load/create_new_group_records.rb

```diff
@@ -0,0 +1,32 @@
+class CreateNewGroupRecords
+  def write(attributes_list)
```

Comment:
KIBA deals with one data file at a time.  The data files we've used thus far are EPUB and JSON.  What's in the data file isn't prescripted by the ETL framework, so one data file can contain multiple records.